Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
```

Comment:
Any reason to put cargo.toml at root directory?

Reply:
yea, it's required to include the C sources into the package (the Cargo.toml in bindings/rust cannot really reference things above it)